CONTRIBUTING.md

```diff
@@ -1,9 +1,32 @@
 ## Contributing
-TypeScript is currently accepting contributions in the form of bug fixes. A bug must have an issue tracking it in the issue tracker that has been approved ("Milestone == Community") by the TypeScript team. Your pull request should include a link to the bug that you are fixing. If you�ve submitted a PR for a bug, please post a comment in the bug to avoid duplication of effort.
 
-New samples will not be accepted at this time.
+## Contributing fixes to existing samples
+
+A bug must have an issue tracking it in the issue tracker that has been approved (labeled "help wanted") by the TypeScript team.
+Your pull request should include a link to the bug that you are fixing.
+If you've submitted a PR for a bug, please post a comment on the issue to avoid duplication of effort.
+
+## Contributing new samples
+
+New samples may be accepted, but will need to first be approved (marked "help wanted" by a TypeScript coordinator with the message "Approved") in the suggestion issue. 
+
+For new samples, please provide a detailed explanation of the intended sample, list of technologies or tools used, and an explanation of why existing samples are not sufficient and a new sample is needed.
+
+A new sample should:
+
+* include a detailed `README.md` file describing how to setup, build, and run the sample.
```

Comment:
What about "include comments explaining why steps are being taken, including in build files if applicable."